alacritty_terminal/src/index.rs

```diff
@@ -44,24 +44,32 @@ impl<L> Point<L> {
 
     #[inline]
     #[must_use = "this returns the result of the operation, without modifying the original"]
-    pub fn sub(mut self, num_cols: usize, length: usize) -> Point<L>
+    pub fn sub(mut self, num_cols: usize, length: usize, absolute_indexing: bool) -> Point<L>
```

Comment:
Could we express this `bool` in function names. I find it a bit odd that function called `sub`/`add` takes a bool now. For e.g. in rust there's a `sub` and a `wrapping_sub`.